README.md

```diff
@@ -7,141 +7,99 @@ Java NETCONF Client.
 
 ## Overview
 
-JNC (Java NETCONF Client) consists of two parts: 
+JNC (Java NETCONF Client) consists of two parts:
 
 * Java library for NETCONF client
-* Pyang  plugin. You need an installation of pyang to use the JNC
-plugin. Get it here: https://github.com/mbj4668/pyang 
-(or use `pip3 install pyang`).
+* pyang plugin for Java class hierarchies generation.
 
-The JNC Pyang plugin can be used to generate Java class hierarchies from YANG data
-models. Together with the JNC library, these generated Java classes may be used
-as the foundation for a NETCONF client (AKA manager) written in Java. 
+You need an installation of *pyang* to use the JNC plugin. Get it [here](https://github.com/mbj4668/pyang) (or use `pip3 install pyang`).
 
-The JNC library is distributed along with the JNC plugin script.
+Together with the JNC library, these generated Java classes may be used as the foundation for a NETCONF client (AKA manager) written in Java.
 
-JNC uses SSHJ (https://github.com/hierynomus/sshj) 
-to communicate with NETCONF servers/agents.
+JNC's Java library code uses [SSHJ](https://github.com/hierynomus/sshj) to communicate with NETCONF servers/agents.
 
 
 ## Getting started
 
+Chapter is divided into following parts:
+
+- [JNC Java library](#jnc-java-library) - describing the Java library for NETCONF clients
+- [JNC pyang plugin](#jnc-pyang-plugin) - dedicated to *pyang* for YANG hierarchy class generation
+- [JNC Java application](#jnc-java-application) - for "final" step of connecting the two tools into combined client application
```

Comment:
I don't think this part is needed. I'd suggest full ToC at the beginning, or omit it completely - note that GitHub generates a ToC, it is just somewhat hidden.

Reply:
I added this primarily as "agenda" to know that we have three steps in the whole process.
I added links as a bonus, and think that at least one intro sentence mentioning three steps is good to have.
If it feels "incomplete",  I can remove the links or add full TOC...

Reply:
I understand the intent now. My point is that in this form it feels like an incomplete ToC in the middle of the document; so maybe change the introductory sentence so that it talks about how to build the thing, not about how the chapter is divided.

Reply:
updated in latest commit